pkg/virt-operator/kubevirt_test.go

```diff
@@ -75,6 +73,7 @@ import (
 	kubecontroller "kubevirt.io/kubevirt/pkg/controller"
 	"kubevirt.io/kubevirt/pkg/monitoring/rules"
 	"kubevirt.io/kubevirt/pkg/testutils"
+	defaults "kubevirt.io/kubevirt/pkg/virt-operator/components/testing"
```

Comment:
It looks like another test helper was used here and place inside the production code imports.

I suggest we do not add this is bad grouping and place this import in its own group.

Also, it makes little sense to me that the name was aliased it to `defaults`. Could you explain the reason?

Reply:
I removed this package, and added the defaults functions to the test itself.